apps/web_delivery/web/views/error_view.ex

```diff
@@ -0,0 +1,9 @@
+defmodule WebDelivery.ErrorView do
+  use WebDelivery.Web, :view
+
+  def render("404.json", _assigns), do: "Not found"
+
+  def render("500.json", _assigns), do: "Internal server error"
```

Comment:
That’s a good question. `JSON.parse` is okay with it. According to _RFC-4627_ it’s invalid. According to `ECMA-404` it’s valid. It might be safer to err on the stricter side and return `"{\"error\":\"Not found\"}"` and `"{\"error\":\"Internal server error\"}"`. What do you think?